query/src/test/scala/filodb/query/exec/AggrOverRangeVectorsSpec.scala

```diff
@@ -130,13 +130,31 @@ class AggrOverRangeVectorsSpec extends RawDataWindowingSpec with ScalaFutures {
     compareIter(result8(0).rows.map(_.getDouble(1)), readyToAggr8.map { v =>
       stdvar(v.map(_.getDouble(1)))
     }.iterator)
+
+    // Stddev
+    val agg9 = RowAggregator(AggregationOperator.Stddev, Nil, tvSchema)
+    val resultObs9a = RangeVectorAggregator.mapReduce(agg9, false, Observable.fromIterable(samples), noGrouping)
+    val resultObs9 = RangeVectorAggregator.mapReduce(agg9, true, resultObs9a, rv=>rv.key)
+    val result9 = resultObs9.toListL.runAsync.futureValue
+    result9.size shouldEqual 1
+    result9(0).key shouldEqual noKey
+
+    val readyToAggr9 = samples.toList.map(_.rows.toList).transpose
```

Comment:
can we use a debox.Buffer here instead of a List since samples will be boxed. Needed for stdvar and stddev methods as well.

Reply:
Thanks for the suggestion! I think we should raise a specific PR for this since it covers tests for all aggregation methods.

Reply:
This is a spec... sorry - no need for any change. :) I assumed it is prod code.